frameworks/CSharp/aspnetcore/PlatformBenchmarks/Data/RawDbMySqlConnector.cs

```diff
@@ -0,0 +1,268 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Caching.Memory;
+using MySqlConnector;
+
+namespace PlatformBenchmarks
+{
+    // Is semantically identical to RawDbNpgsql.
+    // If you are changing RawDbMySqlConnector, also consider changing RawDbNpgsql.
+    public class RawDbMySqlConnector : IDb
+    {
+        private readonly ConcurrentRandom _random;
+        private readonly string _connectionString;
+        private readonly MemoryCache _cache = new MemoryCache(
+            new MemoryCacheOptions()
+            {
+                ExpirationScanFrequency = TimeSpan.FromMinutes(60)
+            });
+
+        public RawDbMySqlConnector(ConcurrentRandom random, AppSettings appSettings)
+        {
+            _random = random;
+            _connectionString = appSettings.ConnectionString;
+        }
+
+        public async Task<World> LoadSingleQueryRow()
+        {
+            using (var db = new MySqlConnection(_connectionString))
```

Comment:
I kept the code is similar as possible to the Npgsql implementation, so that general changes to one class can be easily copied over to the other implementation, as we are doing copy-paste-inheritance here.

So if we modernize one class, we should do the same to the other, but I did not want to introduce anything unnecessary in regards to the point of this commit, so it is clear what is responsible for a potential performance impact on PostgreSQL.